client/src/components/CollegeShortlist.js

```diff
@@ -0,0 +1,103 @@
+        this.closeCollegeModal = this.closeCollegeModal.bind(this);
+        var shortlist = this.shortlist.bind(this);
+    }
+
+    openCollegeModal() {
+        this.setState({
+            collegeModalOpen: true
+        })
+    }
+    closeCollegeModal() {
+        this.setState({
+            collegeModalOpen: false
+        }, () => {
+            this.props.refreshProfile(STUDENT, this.props.details._id)
+        })
+    }
+    shortlist(update) {
+        alert('We pass argument from Child to Parent: ' + update);
+        this.setState({
+            collegeShortlist: update,
+        })
+    }
+
+    render() {
+        const details = this.props.details;
+        const isViewOnly = this.props.isViewOnly;
+        var shortlist = this.shortlist;
+        if (details.collegeShortlist.size > 0) {
+            this.collegeShortlist = details.collegeShortlist.map(e => e.name).join(', ');
+        }
+
+        return (
+            <div>
+            
+            <Menu secondary stackable>
+                <Menu.Item
+                    id='shortlist'
+                    name='College Shortlist'
+                    active={this.state.activeItem === 'shortlist'}
+                    onClick={this.handleMenuClick}
+                >
+                    College Shortlist
+                    {   (this.state.shortlist !== []) && 
+                         <Label color='teal'>{details.collegeShortlist.length}</Label>
+                    }
+                </Menu.Item>
+                
+            </Menu>
+            {this.state.activeItem === 'shortlist' &&
+                <div style={{paddingLeft: 13, paddingRight: 13}}>
+                    <Card fluid>
+                    <Card.Content>
+                    <Card.Header>{details.name || 'Unavailable'}</Card.Header>
+                    {/* <Card.Description>Colleges: {this.collegeShortlist || 'Unavailable'} */}
+                    <Card.Description>Colleges: {details.collegeShortlist.map(e => e.name).join(', ') || 'Unavailable'}
+                    
+                        {
+                            !isViewOnly ? 
+                            <>
+                                <Button
+                                    style={{'margin': '0 0 2px 2px'}}
+                                    icon
+                                    inverted
+                                    size="mini"
+                                    onClick={this.openCollegeModal}
+                                >
+                                    <Icon name='pencil' color="blue"/>
+                                </Button>
+                                <CollegeShortlistModal
+                                    modalOpen={this.state.collegeModalOpen}
+                                    id={details._id}
+                                    collegeShortlist={details.collegeShortlist}
+                                    shortlist = {shortlist.bind(this)}
```

Comment:
you can simply pass in `addToShortlist` as `addToShortlist={this.addToShortlist}`